### PR TITLE
first draft contract API

### PR DESCRIPTION
**Pull request by NikoBergemann**

### CODE REVIEW

File: hlf/chaincode/draft.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+       - Description: This error is returned, if the given parameters could not be parsed. If some attributes are not well formatted, they are listed in invalidParams.
+
+### Get Signatures
+- ID = getCertificate
```
Comment (nklsd) [inline hlf/chaincode/draft.md]:
```suggestion
- ID = getSignatures
```

File: hlf/chaincode/draft.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      ```
+      - Description: This error is returned, if the transaction is not present on the ledger.
+    - [GenericError](errors.md#GenericError) 
+      ```json
+      {
+        "type": "HLUnprocessableLedgerState",
+        "title": "The state on the ledger does not conform to the specified format"
+      }
+      ```
+      - Description: This error is returned, if the state of data on the ledger is not consistent with the curent model. This error should only occurr if the model changes while the old ledger state remains without modification.
```
Comment (nklsd) [inline hlf/chaincode/draft.md]:
```suggestion
      - Description: This error is returned, if the state of data on the ledger is not consistent with the current model. This error should only occur if the model changes while the old ledger state remains without modification.

```

File: hlf/chaincode/draft.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    -
```
Comment (nklsd) [inline hlf/chaincode/draft.md]:
```suggestion

```

File: hlf/chaincode/draft.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+       - Description: This error is returned, if the given parameters could not be parsed. If some attributes are not well formatted, they are listed in invalidParams.
+
+### Get Signatures
```
Comment (nklsd) [inline hlf/chaincode/draft.md]:
```suggestion
### GetSignatures
```

**Review by nklsd — CHANGES_REQUESTED** [review]

**Review by nklsd — APPROVED** [review]

File: hlf/chaincode/draft.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+In the following, the chaincode api for the "Draft"-Chaincode is described.
+This contains all Transactions and Models provided for this domain.
+The Errors returned are defined [here](errors.md#Errors).
+
+## Transactions
+
+### AddOrExtendSignature
+- ID = addOrExtendSignature
+- Send
+    - hash(transactionName, params) :: String
```
Comment (matthias-geuchen) [inline hlf/chaincode/draft.md]: I'd just pass the transactionName and params to the transaction. This way we can do all hashing in chaincode and do not have to worry about that anywhere else.

File: hlf/chaincode/draft.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+      ```
+      - Description: This error is returned, if the state of data on the ledger is not consistent with the current model. This error should only occur if the model changes while the old ledger state remains without modification.
+
+
+## <a id="Models" />Models
+
+### <a id="SignatureData" />SignatureData
+```json
+{
+  "transactionHash": "string",
```
Comment (matthias-geuchen) [inline hlf/chaincode/draft.md]: The transactionHash would then be unnecessary here.

**Review by matthias-geuchen — COMMENTED** [review]
Few things I'd like to discuss

**Review by matthias-geuchen — APPROVED** [review]

**Pull request merged**